providers/attio/connector.go

```diff
@@ -0,0 +1,63 @@
+package attio
+
+import (
+	"github.com/amp-labs/connectors/common"
+	"github.com/amp-labs/connectors/common/paramsbuilder"
+	"github.com/amp-labs/connectors/common/urlbuilder"
+	"github.com/amp-labs/connectors/providers"
+)
+
+const (
+	apiVersion = "/v2"
```

Comment:
Can you truncate the `/` out and use a method like filepath.Join (or if we have some other util function) instead wherever we use it? This is easy to make changes and cause bugs with

Reply:
`urlbuilder` deals with this issues. It is prefered to not have slashes for sure for readability.
```
// Constructor handles path
urlbuilder.New(c.BaseURL, uriPart1, uriPart2, uriPart3, uriPart4)
```